diff -aurN lightdm-1.10.0.orig/data/lightdm.conf lightdm-1.10.0/data/lightdm.conf
--- lightdm-1.10.0.orig/data/lightdm.conf	2014-04-19 12:25:58.758401229 +0200
+++ lightdm-1.10.0/data/lightdm.conf	2014-04-19 12:27:08.214341459 +0200
@@ -29,6 +29,9 @@
 #sessions-directory=/usr/share/lightdm/sessions:/usr/share/xsessions
 #remote-sessions-directory=/usr/share/lightdm/remote-sessions
 #greeters-directory=/usr/share/lightdm/greeters:/usr/share/xgreeters
+greeter-user=xdm
+minimum-vt=1
+run-directory=/run/lightdm
 
 #
 # Seat defaults
@@ -107,6 +110,8 @@
 #autologin-in-background=false
 #autologin-session=UNIMPLEMENTED
 #exit-on-failure=false
+session-wrapper = /etc/lightdm/xsession
+display-setup-script = xrdb -merge -nocpp /etc/lightdm/Xresources
 
 #
 # Seat configuration
diff -aurN lightdm-1.10.0.orig/data/users.conf lightdm-1.10.0/data/users.conf
--- lightdm-1.10.0.orig/data/users.conf	2014-04-19 12:25:58.758401229 +0200
+++ lightdm-1.10.0/data/users.conf	2014-04-19 12:26:28.546186695 +0200
@@ -9,6 +9,6 @@
 # hidden-shells = Shells that indicate a user cannot login
 #
 [UserAccounts]
-minimum-uid=500
+minimum-uid=1000
 hidden-users=nobody nobody4 noaccess
-hidden-shells=/bin/false /usr/sbin/nologin
+hidden-shells=/usr/bin/false /usr/sbin/nologin
