diff -aurN lightdm-1.9.5.orig/data/lightdm.conf lightdm-1.9.5/data/lightdm.conf
--- lightdm-1.9.5.orig/data/lightdm.conf	2013-12-22 22:18:53.243276339 +0100
+++ lightdm-1.9.5/data/lightdm.conf	2013-12-22 22:21:04.365314308 +0100
@@ -17,14 +17,14 @@
 #
 [LightDM]
 #start-default-seat=true
-#greeter-user=lightdm
+greeter-user=xdm
 #minimum-display-number=0
-#minimum-vt=7
+minimum-vt=1
 #lock-memory=true
 #user-authority-in-system-dir=false
 #guest-account-script=guest-account
 #log-directory=/var/log/lightdm
-#run-directory=/var/run/lightdm
+run-directory=/run/lightdm
 #cache-directory=/var/cache/lightdm
 #sessions-directory=/usr/share/lightdm/sessions:/usr/share/xsessions
 #remote-sessions-directory=/usr/share/lightdm/remote-sessions
@@ -93,10 +93,10 @@
 #user-session=default
 #allow-guest=true
 #guest-session=UNIMPLEMENTED
-#session-wrapper=lightdm-session
+session-wrapper=/etc/lightdm/xsession
 #greeter-wrapper=
 #guest-wrapper=
-#display-setup-script=
+display-setup-script=xrdb -merge -nocpp /etc/lightdm/Xresources
 #display-stopped-script=
 #greeter-setup-script=
 #session-setup-script=
diff -aurN lightdm-1.9.5.orig/data/users.conf lightdm-1.9.5/data/users.conf
--- lightdm-1.9.5.orig/data/users.conf	2013-12-22 22:18:53.243276339 +0100
+++ lightdm-1.9.5/data/users.conf	2013-12-22 22:21:42.124765263 +0100
@@ -9,6 +9,6 @@
 # hidden-shells = Shells that indicate a user cannot login
 #
 [UserAccounts]
-minimum-uid=500
+minimum-uid=1000
 hidden-users=nobody nobody4 noaccess
-hidden-shells=/bin/false /usr/sbin/nologin
+hidden-shells=/usr/bin/false /usr/sbin/nologin
