diff -aurN lightdm-1.10.0.orig/data/lightdm.conf lightdm-1.10.0/data/lightdm.conf
--- lightdm-1.10.0.orig/data/lightdm.conf	2014-04-13 21:39:27.935896078 +0200
+++ lightdm-1.10.0/data/lightdm.conf	2014-04-13 21:43:25.851224900 +0200
@@ -29,6 +29,9 @@
 #sessions-directory=/usr/share/lightdm/sessions:/usr/share/xsessions
 #remote-sessions-directory=/usr/share/lightdm/remote-sessions
 #greeters-directory=/usr/share/lightdm/greeters:/usr/share/xgreeters
+greeter-user=xdm
+minimum-vt=1
+run-directory=/run/lightdm
 
 #
 # Seat defaults
@@ -69,7 +72,9 @@
 # autologin-session = Session to load for automatic login (overrides user-session)
 # autologin-in-background = True if autologin session should not be immediately activated
 # exit-on-failure = True if the daemon should exit if this seat fails
-#
+session-wrapper = /etc/lightdm/xsession
+display-setup-script=xrdb -merge -nocpp /etc/lightdm/Xresources
+
 [SeatDefaults]
 #type=xlocal
 #xdg-seat=seat0
diff -aurN lightdm-1.10.0.orig/data/users.conf lightdm-1.10.0/data/users.conf
--- lightdm-1.10.0.orig/data/users.conf	2014-04-13 21:39:27.935896078 +0200
+++ lightdm-1.10.0/data/users.conf	2014-04-13 21:43:54.392286175 +0200
@@ -9,6 +9,6 @@
 # hidden-shells = Shells that indicate a user cannot login
 #
 [UserAccounts]
-minimum-uid=500
+minimum-uid=1000
 hidden-users=nobody nobody4 noaccess
-hidden-shells=/bin/false /usr/sbin/nologin
+hidden-shells=/usr/bin/false /usr/sbin/nologin
